src/packages/application/router/index.spec.tsx

```diff
@@ -61,14 +63,18 @@ describe('RBACLink Component', () => {
 		useLocation.mockReturnValue({ pathname: '/' });
 
 		renderWithAppContext(
-			<Provider store={store}>
-				<RBACLink>
-					<div>Child Component</div>
-				</RBACLink>
-			</Provider>
+			<OidcProvider>
+				<Provider store={store}>
+					<RBACLink>
+						<div>Child Component</div>
+					</RBACLink>
+				</Provider>
+			</OidcProvider>
 		);
 
-		const logoutButton = screen.getByText(D.authentication.logout);
```

Comment:
idem :
`TestingLibraryElementError: Unable to find an accessible element with the role "button" and name "Logout"`